homeassistant/components/netatmo/camera.py

```diff
@@ -132,9 +131,7 @@ def __init__(
 
         self._id = camera_id
         self._home_id = home_id
-        self._device_name = self._data.get_camera(camera_id=camera_id).get(
-            "name", UNKNOWN
-        )
+        self._device_name = self._data.get_camera(camera_id=camera_id)["name"]
```

Comment:
Will the `"name"` key always be present in the dict? The old code looks like we expected it to sometimes be missing.

Reply:
I thought about this too. The Netatmo API schema doesn't mention the camera name as optional. I'll also just opened a PR for pyatmo to ensure the name field is present under all circumstances.